internal/oci/uvm.go

```diff
@@ -128,6 +130,15 @@ func parseAnnotationsUint64(a map[string]string, key string, def uint64) uint64
 	return def
 }
 
+// parseAnnotationsString searches `a` for `key` and if found verifies that the
```

Comment:
Don't need the part about `verifies that the value is a string` in this comment.

Reply:
Agreed.